selinux/Makefile.am

```diff
@@ -0,0 +1,32 @@
+SELINUX_POLICY_FILES = \
+	selinux/cockpit.fc \
+	selinux/cockpit.if \
+	selinux/cockpit.te \
+	$(NULL)
+
+SELINUX_POLICY_MANPAGES = \
+	selinux/cockpit_session_selinux.8cockpit \
+	selinux/cockpit_ws_selinux.8cockpit \
+	$(NULL)
+
+EXTRA_DIST += \
+	$(SELINUX_POLICY_FILES) \
+	$(SELINUX_POLICY_MANPAGES) \
+	$(NULL)
+
+cockpit.pp: $(SELINUX_POLICY_FILES)
+	$(AM_V_GEN) make -sf /usr/share/selinux/devel/Makefile cockpit.pp
+
+cockpit.pp.bz2: cockpit.pp
+	$(AM_V_GEN) bzip2 -9 < $< > $@.tmp && mv $@.tmp $@
+
+selinuxpackagesdir = $(datadir)/selinux/packages/$(SELINUX_POLICY_TYPE)/
+selinuxactivedir = $(sharedstatedir)/selinux/$(SELINUX_POLICY_TYPE)/active/modules/200/cockpit
+
+install-data-hook::
+if SELINUX_POLICY_ENABLED
```

Comment:
This condition feels like on the wrong place. When building without --with-selinux..., it would still build cockpit.pp, and thus require the SELinux makefile and tools. This now adds a new unconditional upstream build dependency, and pointlessly builds the policy just to not install it. I think this entire file should be wrapped into this condition.

Reply:
Unless the files are actually added to targets that make should attempt to build (ie: in the two lines

```
selinuxpackages_DATA = cockpit.pp.bz2
man_MANS += $(SELINUX_POLICY_MANPAGES)
```

both inside the `if`) then nothing will actually call the rules, and nothing will be built.

Of course, you could manually type `make cockpit.pp`, but then that's your business.

As for the oddity of this:

```
install-data-hook::
if SELINUX_POLICY_ENABLED
```

instead of

```
if SELINUX_POLICY_ENABLED
install-data-hook::
```

that's to avoid the silly automake warnings about multiple definitions of the same rule, because automake treats `::` rules as non-portable, and therefore something that ought not to be used in a automake package...

Reply:
I see, so I do understand how it does not get in the way of upstream builds: the only thing that actually depends on cockpit.pp is selinuxpackages_DATA, which is conditionalized by SELINUX_ENABLE. Thanks!